src/webchat-ui/components/plugins/message/regular/index.tsx

```diff
@@ -2,12 +2,15 @@ import * as React from 'react';
 import { MessagePlugin, MessageComponentProps } from '../../../../../common/interfaces/message-plugin';
 import MessageBubble from '../../../presentational/MessageBubble';
 
+const regularMessageBot = 'regular-message-text regular-message-bot-text';
+const regularMessageUser = 'regular-message-text regular-message-user-text';
+
 const RegularMessage = ({ message: { text, source } }: MessageComponentProps) => (
     <MessageBubble
         color={source === 'bot' ? 'primary' : 'default'}
         align={source === 'bot' ? 'left' : 'right'}
         dangerouslySetInnerHTML={{ __html: text || '' }}
-        className={source === 'bot' ? 'webchat-chat-message-bot-text' : 'webchat-chat-message-user-text'}
+        className={source === 'bot' ? regularMessageBot : regularMessageUser}
```

Comment:
this could also contain any kind of HTML, we should omit the "text" in the class name, take `regular-message`

Reply:
fixed